hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/utils/ActionsHelper.java

```diff
@@ -31,12 +32,19 @@ public class ActionsHelper {
      * @return the {@link ContractAction} representing the frame as a call to a missing address
      */
     public ContractAction createSynthActionForMissingAddressIn(@NonNull final MessageFrame frame) {
+        Bytes targetAddress = Bytes.EMPTY;
+        try {
+            targetAddress = tuweniToPbjBytes(frame.getStackItem(1));
+        } catch (final UnderflowException ignored) {
+            // If the stack is too small, just fall through and use empty bytes
```

Comment:
shouldnt we log  minimum the erro  to be aware of these kind of errors even if they are not treated as catastrophic